conf/test_nopreprocessing.config

```diff
@@ -47,6 +47,8 @@ params {
     run_kmcp                               = true
     run_ganon                              = true
     run_krona                              = true
+    run_sylph                              = true
+    sylph_taxonomy                         = params.pipelines_testdata_base_path + 'taxprofiler/data/database/sylph/test_taxonomy.tsv.gz'
```

Comment:
Is there a reason why the `sylph_taxonomy` have been added specifically to this test? 

Now I think about it... should we put all refernces to external data across all test configs (even if we don't run the tool itself), that way if you run a profile but then override a `run_xyz = false` with `--run_xyz true` on testing it should still work? 

We can do this is a separate PR if you think it's a good idea.

Reply:
We need taxonomy to run sylph, so if this tests involves running sylph, then we need it here as well.